testing/dsp_input_sim/dsp_input_sim.cpp

```diff
@@ -178,10 +203,14 @@ void signals(zmq::context_t &context)
       << " after "
       << std::chrono::duration_cast<std::chrono::milliseconds>(shr_end - shr_start).count()
       << "ms" << std::endl;
-
+*/
+    samples_metadata.set_time_zero(0.0);
+    samples_metadata.set_ringbuffer_size(ringbuffer_size);
+    double start_time = (sqn_num + 1) * num_samples / rx_rate;
```

Comment:
Just curious why sqn_num + 1 ?

Reply:
Since we are faking input from the driver, we can't have time_zero and start time be the same, otherwise the calculation into the ringbuffer doesnt make sense. We get around this by shifting one sequence into the future. Its all arbitrary for the simulator purpose anyway. It doesn't really matter where we start indexing. I'll add this as a comment.